source/main/modules/fsal/fsal-file.js

```diff
@@ -222,11 +222,14 @@ function parseFileContents (file, content) {
   while ((match = tagRE.exec(mdWithoutCode)) != null) {
     let tag = match[1]
     tag = tag.replace(/#/g, '') // Prevent headings levels 2-6 from showing up in the tag list
-    if (tag.length > 0) file.tags.push(match[1].toLowerCase())
+    if (tag.length > 0) file.tags.push(match[1].toLocaleLowerCase())
```

Comment:
Great find! But you should provide the user's language for that to work properly (I think it's `global.config.get('app_lang')`)